meson.build

```diff
@@ -343,6 +343,28 @@ if rtaudio_dep.found() == false and jack_dep.found() == false
 	configure.''')
 endif
 
+libsamplerate_dep = []
+found_libsamplerate = false
+if get_option('libsamplerate').allowed()
+	opt_var = cmake.subproject_options()
+	if get_option('buildtype') == 'release'
```

Comment:
This is not a good choice. Distributions use debugoptimized for releases because they generate packages for debug symbols. The wording is just stupid because they copied the naming from cmake.